sktime/forecasting/base/_base.py

```diff
@@ -418,11 +418,17 @@ def predict(self, fh=None, X=None):
 
         Parameters
         ----------
-        fh : int, list, np.array or ``ForecastingHorizon``, optional (default=None)
+        fh : int, list, np.array, pd.Index or ``ForecastingHorizon``,
```

Comment:
I'm okay with this change, but I'll note that I'm not 100% certain if it's really exhaustive types or not if that's the goal.

For example, recently `polars` is added, do you know if `.index` of that is allowed or not? May be @pranavvp16 knows.

Reply:
I think coercible iterables are fine.